src/ray/protobuf/autoscaler.proto

```diff
@@ -0,0 +1,172 @@
+/*
+This file defines an experimental interface designed for Ray
+autoscalers.
+
+It is currently unstable and under active development, but it is
+intended to become stable and forwards-compatible in the future.
+*/
+
+
+syntax = "proto3";
+option cc_enable_arenas = true;
+package ray.rpc;
+
+message ResourceBundle {
+  // The fixed point resource size of the resource (granularity: 1/10000).
+  map<string, int64> resources = 1;
+}
+
+// Type of placement group strategy.
+enum PlacementStrategy {
```

Comment:
task/actor has PlacementStrategy too. i.e. NodeAffinityStrategy. What's our thought here?

Reply:
In the specific case of `NodeAffinityStrategy` (and the other existing strategies), I don't think the autoscaler can really do much with that information, but in general, I think that's a very good question and I don't have a general answer...

Do you have any specific scheduling strategies in mind?

Reply:
Maybe the comment should say "placement strategy" (drop the "group")?

I have not been following the progress on the new placement APIs at all, but is there overlap with the placement groups concepts?
If not, is there potential for useful concept/code overlap?

Reply:
So the SchedulingStrategies are for tasks/actors (with perhaps an overlapping "job to be done" with placement groups). At the moment,

* Spread is soft so we can ignore it while binpakcing
* Node affinity can be filtered within GCS
* Default is default
* Placement group - these require the special case handling.

The most likely thing that could really throw a wrench in this is some soft of Node-anti-affinity strategy, or placement group resizing, or placement group rescheduling.

Reply:
cc @jjyao @rkooo567

Reply:
I think we should send SchedulingStrategies information to autoscaler as well so it has enough information to make a good decision. NodeAntiAffinityStrategy would be interesting and the whole taint and tolerance thing.